nucleus/osgi-platforms/felix/src/main/resources/config/osgi.properties

```diff
@@ -379,10 +379,8 @@ jre-1.6=\
  org.xml.sax.ext, \
  org.xml.sax.helpers, ${endorsed-standard-packages}
 
-#TODO revert the version change in javax.annotation !!!
 endorsed-standard-packages=\
- javax.annotation; version=1.4, \
- javax.annotation.security; version=1.4, \
+ javax.annotation, \
```

Comment:
Needs to be Jakarta now

Reply:
I do not think so. This refers to JDK packages to be allowed for overriding by standard endorsed mechanism within OSGi. I explicitly added version number for GF to manage to resolve jersey bundles. The problem is that while this helps to some extent, Jersey requires Priority annotation which has never made it to JDK/Java SE, so with this hack, jersey is just failing a bit later then it would done without this.